_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5942,6 +5942,8 @@ void MfxHwH264Encode::SetDefaults(
                         ? GetMaxCodedFrameSizeInKB(par)
                         : bufferSizeInBits / 8000;
             }
+            par.calcParam.bufferSizeInKB = IPP_MAX(par.calcParam.bufferSizeInKB, par.calcParam.initialDelayInKB);
```

Comment:
same

Reply:
yep. And that's the reason of build failures.